docs/_sass/swagger.scss

```diff
@@ -76,17 +82,23 @@ h5 {
 
 .swagger-method-name {
   display: flex;
-  width: 100px;
+  width: 50;
```

Comment:
This could benefit from a unit of measurement i.e 50**px** or 3.125**rem**?

Reply:
Sounds real good, yes.